django_project/blog/views.py

```diff
@@ -11,6 +11,7 @@
     CreateView,
     UpdateView,
     DeleteView,
+    View
```

Comment:
I used this View, and not a DeleteView, because the DeleteView requires a template, and I haven't come up with one yet